daml-lf/kv-support/src/test/scala/com/daml/lf/kv/archives/ArchiveConversionsSpec.scala

```diff
@@ -55,8 +61,54 @@ class ArchiveConversionsSpec extends AnyWordSpec with Matchers {
       val archive = Archive.newBuilder().setHash("???").build()
 
       ArchiveConversions.parsePackageIdsAndRawArchives(List(archive)) shouldBe Left(
-        ConversionError.ParseError("non expected character 0x3f in Daml-LF Package ID \"???\"")
+        ArchiveError.Parsing("non expected character 0x3f in Daml-LF Package ID \"???\"")
       )
     }
   }
+
+  "decodePackages" should {
+    "successfully decode packages" in {
+      val List(archive1, archive2) = for {
+        i <- List(1, 2)
```

Comment:
I think we should use a more descriptive naming for this, would be useful to know what 1 and 2 signify

Reply:
It's just a simple differentiator, so I called it like this :)